docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Understand Azure SDK client library method types
+description: Learn about the key differences between Azure SDK client library protocol and convenience methods
+ms.topic: conceptual
+ms.custom: devx-track-dotnet, engagement-fy23, devx-track-arm-template
+ms.date: 06/24/2024
+---
+
+# Azure SDK for .NET protocol and convenience methods overview
+
+The Azure SDK client libraries provide an interface to Azure service REST APIs by translating method calls into underlying HTTP requests. In this article, you'll learn about the different types of methods exposed by the client libraries and explore their implementation patterns.
+
+## Understand protocol and convenience methods
+
+The Azure SDK for .NET client libraries expose two different categories of methods to make requests to an Azure service:
+
+- **Protocol methods** provide a thin wrapper around the underlying REST API for a corresponding Azure service. These methods map primitive input parameters to HTTP request values and return a raw HTTP response object.
+
+- **Convenience methods** provide a convenience layer over the lower-level protocol layer to add support for the .NET type system and other benefits. Convenience methods accept primitives or C# model types as parameters and map them to the body of an underlying REST API request. These methods also handle various details of request and response management to allow developers to focus on sending and receiving data objects, instead of lower-level concerns.
+
+### Azure SDK client library dependency patterns
+
+Protocol and convenience methods implement slightly different patterns based on the underlying package dependency chain of the respective library. The Azure SDK for .NET client libraries depend on one of two different lower-level libraries:
```

Comment:
Might be worth calling them "core" libraries instead of lower-level libraries?  My thinking is that it's true they implement the lower-level part of the stack relative to client libraries, but they also contain convenience layer types like DefaultAzureCredential and other end-user types.